### ISSUE TITLE
Document reset

### ISSUE THREAD

**Issue by zoglesby** [danito/KnownMastodon#12]
Can you please document how to reset this plugin. I had an issues and the initial setup failed and I can not for the life of me figure out how to clear the settings and try again. I have poked around in the database tried removing things from the `config` table, but nothing seems to get it back to a usable state. Anything you can provide would be helpful.

**Comment by danito** [comment]
Hi,

to reset everything, you can delete the "Mastodon" section in your user profile from the user entity
````sql
SELECT *
FROM `entities`
WHERE UUID = 'https://your.known.server/profile/yourusername'
````
In the _contents_ you'll find something like
````json
{ ...
    "mastodon":{"nxd4n@mastodon.social":   
        {"server":"mastodon.server","login":"your@email.address","username":"username","bearer":""}
}
````
(Be careful, this is your user profile, messing this up breaks things).
And also revoke your Known in the authorized applications 
In my case https://mastodon.social/oauth/authorized_applications

**Comment by rikmendes** [comment]
Hmm so if I want to change my instance I have no other but this route to reset the plugin ?

**Comment by jaranta** [comment]
I tried this route for resetting the plugin, but this doesn't seem to work. The mysql command listed above returns an empty set.

**Comment by cdn** [comment]
Alternatively [a branch](https://github.com/cdn/KnownMastodon/tree/admin-server-deletion) on my fork with per account, and server deletion capability

**Comment by cdn** [comment]
That branch has been merged in, so this should be closable @danito?

**Comment by danito** [comment]
fixed by #26
